python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -60,6 +60,12 @@ class NCMECCollabConfig(
     _NCMECCollabConfigRequiredFields,
 ):
     api: str = field(init=False, default=_API_NAME)
+    only_esp_ids: t.Set[int] = field(
+        default_factory=set,
+        metadata={
+            "help": "Only take entries from these eletronic service provider (ESP) ids"
```

Comment:
speels

Reply:
Can you add some help text on how to use for multiple ESP ids? It is not immediately apparent to me whether I should do 

```
--only-esp-ids <esp_1>,<esp_2>

OR

--only-esp-ids <esp_1> --only-esp-ids <esp_2>
```

Reply:
I can add the default hint to mention it's CSV (though I could use the argparse features to allow multiple flags as you put, I think csv is more clear/consistent with other commands)